addons/repair/models/repair.py

```diff
@@ -617,11 +618,12 @@ def constrain_lot_id(self):
         for line in self.filtered(lambda x: x.product_id.tracking != 'none' and not x.lot_id):
             raise ValidationError(_("Serial number is required for operation line with product '%s'") % (line.product_id.name))
 
-    @api.depends('price_unit', 'repair_id', 'product_uom_qty', 'product_id', 'repair_id.invoice_method')
-    def _compute_price_subtotal(self):
+    @api.depends('price_unit', 'repair_id', 'product_uom_qty', 'product_id', 'tax_id', 'repair_id.invoice_method')
+    def _compute_price_total_and_subtotal(self):
```

Comment:
As the PR is in stable you should leave the method signature unchanged. Maybe someone has overridden it

Reply:
Done